py3status/modules/online_status.py

```diff
@@ -3,30 +3,29 @@
 Determine if you have an Internet Connection.
 
 Configuration parameters:
-    cache_timeout: how often to run the check (default 10)
-    format: display format for online_status (default '{icon}')
-    icon_off: what to display when offline (default '■')
-    icon_on: what to display when online (default '●')
-    timeout: how long before deciding we're offline (default 2)
-    url: connect to this url to check the connection status
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default '{icon}')
+    icon_off: show when connection is offline (default '■')
+    icon_on: show when connection is online (default '●')
+    timeout: number of seconds before announcing offline (default 2)
```

Comment:
`timeout: time to wait for a response, in seconds.` ?
`timeout: wait in seconds before deciding offline`  ?
`timeout: seconds to wait before deciding offline` ?
`timeout: specify a timeout, in seconds, before deciding offline` ?